main/test/flix/Test.Exp.Concurrency.NewChannel.flix

```diff
@@ -1,93 +1,93 @@
 namespace Test/Exp/Concurrency/NewChannel {
 
     @test
-    def testNewChannel01(): (Sender[Unit, Static], Receiver[Unit, Static]) \ IO = Channel.unbuffered(Static)
+    def testNewChannel01(): Unit \ IO = region _r { discard Channel.unbuffered(Static) }
```

Comment:
I think for these tests to make sense we should:

- Use the region.
- Put an element of the appropriate type e.g. `Bool`, `String` and so forth.
- Retrieve said element.

Reply:
I was trying to keep the change as small as possible, and follow your earlier guidance that tests should only test the thing they're intended to test (i.e. these tests are supposedly only testing `newChannel`, not `send` and `receive` which are tested elsewhere).

Reply:
True, but now all the tests are the same, right?

Reply:
I think the minimum delta would be to:

- Introduce region.
- Create channel in a let with a type, e.g. `let _: Channel[....]  = ...`.

Reply:
But I think my first outline (which admittedly requires the tests to be rewritten) would be best.